terraform/modules/marklogic/marklogic_cf_template.yml

```diff
@@ -767,4 +767,4 @@ Resources:
         IamInstanceProfile:
           Name: !Ref IAMRole
         MetadataOptions:
-          HttpTokens: optional
```

Comment:
Dunno if it matters but since terraform is normally funny about line endings FYI

Reply:
In this case it's to stop Terraform trying to always change this file, at least for me